src/core/eme/eme_manager.ts

```diff
@@ -209,9 +207,15 @@ export default function EMEManager(
           const { mediaKeySession,
                   sessionType } = sessionEvt.value;
 
+          // `generateKeyRequest` awaits a single Uint8Array containing all
+          // initialization data.
+          const concatInitData = concat(...initializationData.values.map(i => i.data));
```

Comment:
I was wondering, as there are some places in the code where the concatenated init data is needed, we could maybe integrate a function in the `initializationData` that returns it, no?

Reply:
Actually there are two places:
  - here, because ultimately the `generateRequest` EME API requires it under that format
  - in the `PersistentSessionsStore`, because that concatenation's has was previously used as an ID to identify persisted sessions.

So they are needed at two places but for two very different reasons IMO. For example the first could be in any order whereas the second has to follow a specific order.

As such, I don't know if I want to use a common function here. Being explicit in the `PersistentSessionsStore` may be clearer.